Cargo.toml

```diff
@@ -463,12 +463,12 @@ http = "1.0"
 http-body = "1.0"
 
 # crypto
-secp256k1 = { version = "0.28", default-features = false, features = [
+secp256k1 = { version = "0.29", default-features = false, features = [
     "global-context",
     "recovery",
 ] }
 # TODO: Remove `k256` feature: https://github.com/sigp/enr/pull/74
-enr = { version = "0.12.0", default-features = false, features = [
+enr = { version = "0.12.1", default-features = false, features = [
```

Comment:
can we roll this back
I'd like to send this directly to main

Reply:
sg